financial-services-accelerator/accelerators/fs-is/carbon-home/dbscripts/financial-services/consent/h2.sql

```diff
@@ -64,39 +64,20 @@ CREATE TABLE IF NOT EXISTS OB_CONSENT_STATUS_AUDIT (
   ACTION_BY         VARCHAR(255),
   PREVIOUS_STATUS   VARCHAR(255),
   PRIMARY KEY(STATUS_AUDIT_ID),
-  CONSTRAINT FK_OB_CONSENT_STATUS_AUDIT FOREIGN KEY (CONSENT_ID) REFERENCES OB_CONSENT (CONSENT_ID)
+  CONSTRAINT FK_FS_CONSENT_STATUS_AUDIT FOREIGN KEY (CONSENT_ID) REFERENCES FS_CONSENT (CONSENT_ID)
 );
 
-CREATE TABLE IF NOT EXISTS OB_CONSENT_FILE (
+CREATE TABLE IF NOT EXISTS FS_CONSENT_FILE (
   CONSENT_ID        VARCHAR(255) NOT NULL,
   CONSENT_FILE      CLOB,
   PRIMARY KEY(CONSENT_ID),
-  CONSTRAINT FK_OB_CONSENT_FILE FOREIGN KEY (CONSENT_ID) REFERENCES OB_CONSENT (CONSENT_ID)
+  CONSTRAINT FK_FS_CONSENT_FILE FOREIGN KEY (CONSENT_ID) REFERENCES FS_CONSENT (CONSENT_ID)
 );
 
-CREATE TABLE IF NOT EXISTS OB_CONSENT_ATTRIBUTE (
+CREATE TABLE IF NOT EXISTS FS_CONSENT_ATTRIBUTE (
   CONSENT_ID        VARCHAR(255) NOT NULL,
   ATT_KEY           VARCHAR(255) NOT NULL,
   ATT_VALUE         VARCHAR(255) NOT NULL,
   PRIMARY KEY(CONSENT_ID, ATT_KEY),
-  CONSTRAINT FK_OB_CONSENT_ATTRIBUTE FOREIGN KEY (CONSENT_ID) REFERENCES OB_CONSENT (CONSENT_ID)
-);
-
-CREATE TABLE IF NOT EXISTS OB_THROTTLE_DATA (
```

Comment:
What is the reason to remove these tables?

Reply:
We are not planning to add these features to Accelerator 4